course/models.py

```diff
@@ -450,6 +450,11 @@ def __str__(self):
     def clean(self):
         super().clean()
         errors = {}
+        RESERVED = ("instances",)
+        if self.instance_name in RESERVED:
+            errors['instance_name'] = _("You cannot use word '{}' as an instance name.").format(self.instance_name)
+        if self.url in RESERVED:
+            errors['url'] = _("You cannot use word '{}' in the url.").format(self.instance_name)
```

Comment:
Of course, the url error message should include `self.url` instead of `self.instance_name`.